diff -urN aaa/mozilla-bonobo-0.3.0/configure.in mozilla-bonobo-0.3.0/configure.in
--- aaa/mozilla-bonobo-0.3.0/configure.in	2003-05-19 20:43:03.000000000 +0200
+++ mozilla-bonobo-0.3.0/configure.in	2003-06-27 21:54:25.000000000 +0200
@@ -13,13 +13,6 @@
 AC_HEADER_STDC
 AM_PROG_LIBTOOL
 
-dnl Checks for programs.
-AC_CHECK_PROG(has_mozilla_config, mozilla-config, yes, no,,)
-if test $has_mozilla_config != yes; then
-   echo "You may need to install the mozilla development package.";
-   exit 1;
-fi
-
 AC_CHECK_PROG(has_pkg_config, pkg-config, yes, no,,)
 if test $has_pkg_config != yes; then
    echo "You need to install pkg-config.";
@@ -29,7 +22,7 @@
 
 dnl Checks for libraries.
 echo -n "Checking for mozilla ldflags... "
-PLUGIN_LDFLAGS=$(mozilla-config --libs)
+PLUGIN_LDFLAGS=$(pkg-config --libs mozilla-plugin)
 AC_SUBST(PLUGIN_LDFLAGS)
 
 echo -n "Checking for GNOME ldflags... "
@@ -39,7 +32,7 @@
 
 dnl Checks for header files.
 echo -n "Checking for mozilla cflags... "
-PLUGIN_CFLAGS=$(mozilla-config --cflags java plugin)
+PLUGIN_CFLAGS=$(pkg-config --cflags mozilla-plugin)
 AC_SUBST(PLUGIN_CFLAGS)
 
 echo -n "Checking for GNOME cflags... "
diff -urN aaa/mozilla-bonobo-0.3.0/src/mozilla-bonobo-plugin.c mozilla-bonobo-0.3.0/src/mozilla-bonobo-plugin.c
--- aaa/mozilla-bonobo-0.3.0/src/mozilla-bonobo-plugin.c	2003-05-19 20:56:40.000000000 +0200
+++ mozilla-bonobo-0.3.0/src/mozilla-bonobo-plugin.c	2003-06-27 22:10:49.000000000 +0200
@@ -23,8 +23,8 @@
  * USA
  */
 
-#include <plugin/npapi.h>
-#include <plugin/npupp.h>
+#include <npapi.h>
+#include <npupp.h>
 
 #include <string.h>
 #include <stdio.h>
